Dynatrace/dynatrace-operator
# Collect support archive output to file.

# Description
Collect all output that the support archive command produces into a file and add it to the support archive.

## How can this be tested?
Build and deploy the operator, run the `support-archive` command. The produced support archive should contain a file called `supportarchive_console.log`.

## Checklist
